### PR TITLE
Added Kristen Osborn and Nicole Smith to the site

### PR DESCRIPTION
**Pull request by ghost**
*If the PR is not a blog post feel free to delete this template, otherwise ensure you complete the following before this PR is approved or merged:*

- [ X ] I have set an appropriate description

### CODE REVIEW

File: src/_data/team.yml [hunk 1]
```diff
@@ -1084,3 +1084,27 @@
...
+    <p>This has involved a combination of service redesign, business process re-engineering and technical delivery. Recent projects have included implementing a new Azure/PowerBI data warehouse platform for Manchester Met University, and leading a customer experience transformation programme at Trafford Council.</p>
+    <p>Kirsten is also studying for an MBA as part of a senior leader apprenticeship at Manchester Met.</p>
+    <p>In her spare time she enjoys road cycling, jazz swing dancing and going to tech meet-ups.</p>
+  email: kirsten.osborn@codurance.com
+  office: Manchester
+
+- id: nicole_smith
+  name: Nicole Smith
+  image: /assets/custom/img/team/nicole_smith.jpg
+  role: New Business Manager
```
Comment (ghost) [inline src/_data/team.yml:1103]: Is it New Business Manager or just Business Manager?

Comment (ghost) [reply src/_data/team.yml:1103]: Screenshot 2020-09-29 at 14 43 35

New Business Manager as per Magda 👍

**Review by ghost — APPROVED** [review]
LGTM

**Comment by sylvester-loreto-codurance** [comment]
No objections at the time of merge

**Pull request merged**